crates/llvm-context/src/polkavm/context/mod.rs

```diff
@@ -666,6 +666,18 @@ where
             .into())
     }
 
+    /// Load bytes32 at given pointer
+    pub fn build_load_bytes(
```

Comment:
This helper is not needed. It boils down to just `self.build_byte_swap(self.build_load(pointer, "bytes_pointer")?)` since bytes32 size equals the word size?